docker-run

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+# This file is a part of IoT-LAB gateway_code
+# Copyright (C) 2015 INRIA (Contact: admin@iot-lab.info)
+# Contributor(s) : see AUTHORS file
+#
+# This software is governed by the CeCILL license under French law
+# and abiding by the rules of distribution of free software.  You can  use,
+# modify and/ or redistribute the software under the terms of the CeCILL
+# license as circulated by CEA, CNRS and INRIA at the following URL
+# http://www.cecill.info.
+#
+# As a counterpart to the access to the source code and  rights to copy,
+# modify and redistribute granted by the license, users are provided only
+# with a limited warranty  and the software's author,  the holder of the
+# economic rights,  and the successive licensors  have only  limited
+# liability.
+#
+# The fact that you are presently reading this means that you have had
+# knowledge of the CeCILL license and that you accept its terms.
+
+# Use Docker-run
+# to run the docker container built by Docker-build.sh
+
+import argparse
+from collections import namedtuple
+
+import os
```

Comment:
This should be the first import. Apart from that, this file is full of pep8 and is not using common practices regarding a python script directly run from cli. You should use:
```python
if __name__ == '__main__':
   ...
```
at the end of the file and maybe define some utility functions.

Reply:
OK mostly taken into account in commit 15e59781a00024da7a5d2878be3f92e6eb2214b7